lmnet/lmnet/utils/predict_output/writer.py

```diff
@@ -0,0 +1,103 @@
+import os
+
+import numpy as np
+
+from lmnet.utils.predict_output.output import ImageFromJson
+from lmnet.utils.predict_output.output import JsonOutput
+
+
+class OutputWriter():
+    def __init__(self, task, classes, image_size, data_format):
+        self.json_output = JsonOutput(task, classes, image_size, data_format)
+        self.image_from_json = ImageFromJson(task, classes, image_size)
+
+    def write(self, dest, outputs, raw_images, image_files, step, save_material=True):
+        """Save predict output to disk.
+           numpy array, JSON, and images if you want.
+
+        Args:
+            dest (str): paht to save file
+            outputs (np.ndarray): save ndarray
+            raw_images (np.ndarray): image ndarray
+            image_files (list[str]): list of file names.
+            step (int): value of training step
+            save_material (bool, optional): save materials or not. Defaults to True.
+        """
+        save_npy(dest, outputs, step)
+
+        json = self.json_output(outputs, raw_images, image_files)
+        save_json(dest, json, step)
+
+        if save_material:
+            materials = self.image_from_json(json, raw_images, image_files)
+            save_materials(dest, materials, step)
+
+
+def save_npy(dest, outputs, step):
```

Comment:
Why these function are not method of `OutputWriter` ?

Reply:
I prefer "Don't put in a method if you don't need it" mentality.
These functions don't need the state of `OutputWriter`.
IMO, This makes these functions independent and easier to test.

Reply:
I see.